src/config/ConfigDescriptions.hpp

```diff
@@ -453,6 +453,12 @@ inline static const std::vector<SConfigOptionDescription> CONFIG_OPTIONS = {
         .type        = CONFIG_OPTION_INT,
         .data        = SConfigOptionDescription::SRangeData{1, 0, 3},
     },
+    SConfigOptionDescription{
+        .value       = "input:focus_on_close",
+        .description = "Controls the window focus behavior when a window is closed. When set to 0, focus will shift to the next window candidate. When set to 1, focus will shift to the window under the cursor. [0/1]",
+        .type        = CONFIG_OPTION_INT,
+        .data        = SConfigOptionDescription::SRangeData{0, 0, 1},
```

Comment:
data type should be CHOICE

Reply:
CHOICE cannot have SRangeData, it has to have SChoiceData

Reply:
yes, looks fine.

these are descriptions that can be queried by `hyprctl descriptions` and used by settings daemons / GUIs